src/components/UserProfile.js

```diff
@@ -96,88 +108,100 @@ function UserProfile() {
-
-                                        </button>
-
-                                        {deleteWarning ? (
-                                            <DeleteConfirmation
-                                            setDeleteWarning={setDeleteWarning}
-                                            handleDelete={handleDelete}
-                                            deleteID={deleteID}
-                                            />
-                                        ) : null}
-
-                                        <button
-                                        className="likeBtn"
-                                        onClick={() => {
-                                            handleLike(result.key, result.likes);
-                                        }}>
-                                            <span className="sr-only">Like</span>
-                                        </button>
-                                        <p className="likeCount">{result.likes}</p>
-                                    </div>
-                                    <h3>{result.userInput}</h3>
-                                    {
-                                        result.results.map((initialWord, index) => {
-                                            return <p key={`${result.key}-${index}`}>{initialWord}</p>;
-                                        })
-                                    }
-                                </li> : ""
-                            )
-                        }) : <ErrorPage />
-                    }
-                </ul>
-
-                <div className='profileButtons'>
-                    <Link to="/">
-                        <button className='backButton'>Back</button>
-                    </Link>
-                    <button className='deleteProfileButton' onClick={() => setDeleteAccountAttempt(true)}
-                    >Delete Account</button>
-                </div>
-            </div>
-        </section>
+                    <div className="wrapper">
+                        <h2>Your Profile</h2>
+                        {
+                            uid === user.uid && backronymKeys.length === 0 ?
+                            <div className="emptyProfileMessage">
+                                <h3>Oops! Looks like you don't have any backronyms created. Go back to the main page to get started!</h3>
+                            </div>
+                            : null
+                        }
+
+                        <ul className="resultsDisplay">
+                            {
+                                uid === user.uid ?
+                                gallery.map((result) => {
+                                    return(
+                                        (result.email === user.email || (result.email === 'anonymous' && user.email === null)) ?
+                                        <li className="galleryCard" key={result.key}>
```

Comment:
I believe a large portion of the code is wrapped inside the GalleryCard component right now to avoid repetitive code with the gallery

Reply:
This is probably what's causing the conflict :(